app/models/upload.rb

```diff
@@ -63,7 +63,9 @@ def strip_chars_from_headers
   end
 
   def self.last_uploads
-    Upload.select('DISTINCT ON("csv_type") *').where(ok: true).order(csv_type: :asc).order(updated_at: :desc)
+    Upload.select('DISTINCT ON("csv_type") *')
+          .order(csv_type: :asc)
```

Comment:
You don't care about `ok: true` anymore?

Reply:
reverted